app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    command_user = command.split(" ")
+
+    if len(command_user) < 3:
```

Comment:
The input validation should ensure that the command contains exactly three elements. Currently, it only checks if there are at least three elements. Consider changing the condition to `if len(command_user) != 3:` to ensure the command is in the correct format.